### DISCUSSION TITLE
Most discussed articles

### DISCUSSION THREAD

**Discussion by goran-zdjelar** [dailydotdev/daily#1113] [Feature Requests]
I've been offline for some time I've noticed that you removed "The most discussed" page. Now there's this "Discussions" page which is a bunch non-related random comments which are totally useless and this page doesn't add any value whatsoever.

**Comment by idoshamun** [comment]
Hi @goran-zdjelar,

Did you use the "most discussed" feed before?
We're aware that right now the discussion feed may not make sense to some users and we will iterate on that to improve. Essentially we believe that comments on their own provide value and can be interesting to scroll through. We just need to figure out how to surface only the good ones

**Reply by goran-zdjelar** [reply]
Yes, I did use "most discussed" feed before and I used it to find articles worth reading. Very often I found better articles there than in "most upvoted" feed, because, if you think about it, it takes more effort to write a comment than to upvote something, and if something has a lot of comments it means it's worth checking out as people are having discussion about the article. 

Regarding the page with comments on their own, I will probably never click that link as I don't use daily dev to find comments but to find interesting articles. But okay, my opinion does not represent the opinion of majority. Maybe you can have both feeds and set up tracking which feed will be more used.

Anyhow, I would suggest to bring back "most discussed" feed as it was very useful feature.
